### ISSUE TITLE
make error problem 

### ISSUE THREAD

**Issue by tron19920125** [s9xie/hed#21]
hi there
i follow the tutorial using the file provided by hed to modify my caffe.
i type make clean,and the make all
some error arise
`CXX src/caffe/layers/filter_layer.cpp
CXX src/caffe/layers/flatten_layer.cpp
CXX src/caffe/layers/hdf5_data_layer.cpp
CXX src/caffe/layers/hdf5_output_layer.cpp
CXX src/caffe/layers/hinge_loss_layer.cpp
src/caffe/layers/bias_layer.cpp: In member function ‘virtual void caffe::BiasLayer<Dtype>::LayerSetUp(const std::vector<caffe::Blob<Dtype>*>&, const std::vector<caffe::Blob<Dtype>*>&)’:
src/caffe/layers/bias_layer.cpp:16:11: error: ‘BiasParameter’ does not name a type
     const BiasParameter& param = this->layer_param_.bias_param();
           ^
src/caffe/layers/bias_layer.cpp:17:52: error: ‘param’ was not declared in this scope
     const int axis = bottom[0]->CanonicalAxisIndex(param.axis());
                                                    ^
src/caffe/layers/bias_layer.cpp: In member function ‘virtual void caffe::BiasLayer<Dtype>::Reshape(const std::vector<caffe::Blob<Dtype>*>&, const std::vector<caffe::Blob<Dtype>*>&)’:
src/caffe/layers/bias_layer.cpp:42:9: error: ‘BiasParameter’ does not name a type
   const BiasParameter& param = this->layer_param_.bias_param();
         ^
src/caffe/layers/bias_layer.cpp:49:41: error: ‘param’ was not declared in this scope
       0 : bottom[0]->CanonicalAxisIndex(param.axis());
                                         ^
make: *** [.build_release/src/caffe/layers/bias_layer.o] Error 1
make: *** Waiting for unfinished jobs....
src/caffe/layers/batch_norm_layer.cpp: In member function ‘virtual void caffe::BatchNormLayer<Dtype>::LayerSetUp(const std::vector<caffe::Blob<Dtype>*>&, const std::vector<caffe::Blob<Dtype>*>&)’:
src/caffe/layers/batch_norm_layer.cpp:12:3: error: ‘BatchNormParameter’ was not declared in this scope
   BatchNormParameter param = this->layer_param_.batch_norm_param();
   ^
src/caffe/layers/batch_norm_layer.cpp:13:30: error: ‘param’ was not declared in this scope
   moving_average_fraction_ = param.moving_average_fraction();
                              ^
make: *** [.build_release/src/caffe/layers/batch_norm_layer.o] Error 1
src/caffe/layers/elu_layer.cpp: In instantiation of ‘void caffe::ELULayer<Dtype>::Forward_cpu(const std::vector<caffe::Blob<Dtype>*>&, const std::vector<caffe::Blob<Dtype>*>&) [with Dtype = float]’:
src/caffe/layers/elu_layer.cpp:44:1:   required from here
src/caffe/layers/elu_layer.cpp:14:54: error: ‘class caffe::LayerParameter’ has no member named ‘elu_param’
   Dtype alpha = this->layer_param_.elu_param().alpha();
                                                      ^
src/caffe/layers/elu_layer.cpp: In instantiation of ‘void caffe::ELULayer<Dtype>::Backward_cpu(const std::vector<caffe::Blob<Dtype>*>&, const std::vector<bool>&, const std::vector<caffe::Blob<Dtype>*>&) [with Dtype = float]’:
src/caffe/layers/elu_layer.cpp:44:1:   required from here
src/caffe/layers/elu_layer.cpp:31:56: error: ‘class caffe::LayerParameter’ has no member named ‘elu_param’
     Dtype alpha = this->layer_param_.elu_param().alpha();
                                                        ^
src/caffe/layers/elu_layer.cpp: In instantiation of ‘void caffe::ELULayer<Dtype>::Forward_cpu(const std::vector<caffe::Blob<Dtype>*>&, const std::vector<caffe::Blob<Dtype>*>&) [with Dtype = double]’:
src/caffe/layers/elu_layer.cpp:44:1:   required from here
src/caffe/layers/elu_layer.cpp:14:54: error: ‘class caffe::LayerParameter’ has no member named ‘elu_param’
   Dtype alpha = this->layer_param_.elu_param().alpha();
                                                      ^
src/caffe/layers/elu_layer.cpp: In instantiation of ‘void caffe::ELULayer<Dtype>::Backward_cpu(const std::vector<caffe::Blob<Dtype>*>&, const std::vector<bool>&, const std::vector<caffe::Blob<Dtype>*>&) [with Dtype = double]’:
src/caffe/layers/elu_layer.cpp:44:1:   required from here
src/caffe/layers/elu_layer.cpp:31:56: error: ‘class caffe::LayerParameter’ has no member named ‘elu_param’
     Dtype alpha = this->layer_param_.elu_param().alpha();
                                                        ^
make: *** [.build_release/src/caffe/layers/elu_layer.o] Error 1`

so what happened?how to  solve such problem？

**Comment by lixiang-ucas** [comment]
i have encountered the same problem, have you fixed it?
